main.nf

```diff
@@ -226,7 +226,7 @@ if (!sdrf_file)
 {
   ch_spectra = Channel.fromPath(spectra_files, checkIfExists: true)
   ch_spectra
```

Comment:
Is there no built in function?

Reply:
unfortunately not that I know of (besides including additional dependencies like apache commons).